<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PubmedSearchDialog</class>
 <widget class="QDialog" name="PubmedSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search on Pubmed ...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="articletools.qrc">
    <normaloff>:/articletools/pubmedsearch.png</normaloff>:/articletools/pubmedsearch.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,3,0">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string> Search ... </string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>phrase:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QComboBox" name="cmbFields2">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>phrase:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="cmbAND1">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AND</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OR</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="edtPhrase2">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="5">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>max. results:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinMaxResults">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>200</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnSearch">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;Search</string>
          </property>
          <property name="icon">
           <iconset resource="articletools.qrc">
            <normaloff>:/articletools/search_active.png</normaloff>:/articletools/search_active.png</iconset>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>in:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QComboBox" name="cmbFields1"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="edtPhrase1"/>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>catalog:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QComboBox" name="cmbCatalog">
        <item>
         <property name="text">
          <string>Pubmed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pubmed Books</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>in:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="5">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item row="1" column="0" colspan="2">
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>status:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labStatus">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QListWidget" name="lstLog">
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>select the records to insert:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="listView">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnAgain">
       <property name="text">
        <string>discard &amp; search again</string>
       </property>
       <property name="icon">
        <iconset resource="articletools.qrc">
         <normaloff>:/articletools/search_disabled.png</normaloff>:/articletools/search_disabled.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>add selected</string>
       </property>
       <property name="icon">
        <iconset resource="../../application/litsoz3.qrc">
         <normaloff>:/add.png</normaloff>:/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../application/litsoz3.qrc"/>
  <include location="articletools.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>PubmedSearchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>577</x>
     <y>620</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>606</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PubmedSearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>666</x>
     <y>616</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>616</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
